scripts/correctionlib_comparison.py

```diff
@@ -0,0 +1,401 @@
+import argparse
+import gzip
+import json
+import os
+from collections import defaultdict
+from itertools import combinations, count, product
+from typing import Any, Dict, Generator, List, Tuple, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import mplhep as hep
+import numpy as np
+
+matplotlib.use("Agg")
+hep.style.use("CMS")
+
+parser = argparse.ArgumentParser(description="Plotting comparison of two correction libs")
+parser.add_argument("--input_a", type=str, help="json.gz correctionlib file A")
+parser.add_argument("--input_b", type=str, help="json.gz correctionlib file B")
+parser.add_argument("--tag_a", type=str, help="Name (str) A", default="A")
```

Comment:
add a bit more verbose help to signal that this is the name to be plotted

Reply:
Added in [0887179](https://github.com/KIT-CMS/TauAnalysis-CROWN/commit/0887179c5d37b3f287597f9342d1aacb647ce35d)

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L17-L29

Reply:
Also in the same commit: a more descriptive output:

https://github.com/KIT-CMS/TauAnalysis-CROWN/blob/0887179c5d37b3f287597f9342d1aacb647ce35d/scripts/correctionlib_comparison.py#L407-L414